02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ZOOlingual Integration
+taxonomy:
+    category: docs
+---
+
+On multilanguage sites, which have both ZOOmailing and ZOOlingual, newsletters preview and sending are available in the language the admin user specifies when he/she logs in the Joomla administrator.
```

Comment:
This part is not clear "the admin user specifies when he/she logs in the Joomla administrator"